lib/crawler/keyword/schemas/keyword.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Crawler.Keyword.Schemas.Keyword do
+  use Ecto.Schema
+
+  import Ecto.Changeset
+
+  schema "keywords" do
+    field :name, :string
+    field :status, Ecto.Enum, values: [:new, :in_progress, :completed, :failed], default: :new
+
+    belongs_to :user, Crawler.Account.Schemas.User
```

Comment:
let alias the `Crawler.Account.Schemas.User` @vnntsu 

```
alias Crawler.Account.Schemas.User

belongs_to :user, User
```

Reply:
I think we can omit the alias since using it only one time. If we use it twice or more in the future, I will add an alias. What do you think?

Reply:
Normally we alias all external module, doesn't matter how many time we use it @vnntsu

Reply:
Fixed in [b997d39](https://github.com/vnntsu/elixir-ic/pull/36/commits/b997d39ec9d7623c77e82f7488a8b8c457e95bc6)